R/RestRserveRequest.R

```diff
@@ -74,6 +82,7 @@ RestRserveRequest = R6::R6Class(
       self$body = body
       self$content_type = content_type
       self$request_id = uuid::UUIDgenerate(TRUE)
+      self$path_parameters = list()
```

Comment:
List not env?

Reply:
lists look fine - no penalty on small objects:
```
> lst = list('a' = 'b')
> env = new.env(parent = emptyenv())
> env[['a']] = 'b'
> microbenchmark::microbenchmark(lst[['a']], env[['a']])
Unit: nanoseconds
       expr min    lq   mean median  uq  max neval
 lst[["a"]] 157 172.0 225.96    178 186 4183   100
 env[["a"]] 140 147.5 176.57    156 161 1555   100
```

Reply:
Sure. By other side, result of the `parse_vars` can be wrapped with `list2env`. We can do it later if we see fit.